src/main/scad/laszlo-molnar/l-i-vator-i/molnar.l-i-vator-i.scad

```diff
@@ -0,0 +1,86 @@
+include <puzzlecad.scad>
+
+box_puzzle_border = 6;
+
+$burr_inset = 0.125;
+$burr_bevel = 1;
+$unit_beveled = true;
+$burr_scale = 17;
+$joint_inset = 0.06;
+
+dim = $burr_scale * 3 + box_puzzle_border * 2 + $burr_inset * 2;
+height = $burr_scale * 3 + box_puzzle_border + $burr_inset * 2;
+
+mid = box_puzzle_border / 2 + 0.5;
+far = dim - mid;
+    
+*box();
+*pieces();
+*cap();
+
+module box() {
+    render(convexity = 2)
+    difference() {
+        beveled_cube([dim, dim, height]);
+        translate([box_puzzle_border, box_puzzle_border, box_puzzle_border])
+        cube([
+            $burr_scale * 3 + $burr_inset * 2,
+            $burr_scale * 3 + $burr_inset * 2,
+            $burr_scale * 4
+        ]);
+     }
+     
+     translate([0, 0, height])
+     connector(1.8, 2.0);
+}
+
+module pieces() {
+    burr_plate([
+        ["xx{connect=mz+x+,clabel=Cy+}"],
```

Comment:
Since you're using oriented connectors (which is great), you can omit the orientation on the labels, e.g., this can be just

{connect=mz+x+,clabel=C}

This is preferred, as puzzlecad will generate warnings for the spurious label orientations (which are ignored if the connector itself is oriented).